src/jvm_wrapper/registration/kt_property.cpp

```diff
@@ -67,17 +67,3 @@ void KtProperty::call_set(jni::Env& p_env, KtObject* instance, const Variant& p_
     jvalue args[1] = {jni::to_jni_arg(instance->get_wrapped())};
     wrapped.call_void_method(p_env, CALL_SET, args);
 }
-
-#ifdef TOOLS_ENABLED
-void KtProperty::safe_call_get(jni::Env& p_env, KtObject* instance, Variant& r_ret) {
```

Comment:
why method is removed from cpp but not from header ?

Reply:
mistake